showcase/go.mod

```diff
@@ -11,3 +11,5 @@ require (
 	google.golang.org/genproto v0.0.0-20201201144952-b05cb90ed32e
 	google.golang.org/grpc v1.33.2
 )
+
+replace github.com/googleapis/gapic-showcase => ./gen/github.com/googleapis/gapic-showcase
```

Comment:
Please revert this. Running `make clean` is a good thing to do before pushing changes.

Reply:
The `go.mod` and `go.sum` changes sneak in under the radar. Would it be correct to add them to `.gitignore`?

Reply:
I'd prefer that we don't add them to the `.gitignore` because this is a submodule and we really do want changes to this file to be captured...except for this one made by the testing script :)

You could add some command to the `showcase.bash` to revert this change once the tests are done though, so that `make clean` isn't necessary.